exampleConfig.js

```diff
@@ -0,0 +1,15 @@
+/*
+Example StatsD configuration for sending metrics to Instrumental.
+See the offical Statsd exampleConfig.js for more StatsD options.
+*/
+{
+ port: 8125
+, backends: ["statsd-instrumental-backend" ]
+, debug: false
+, instrumental: {
+    key: "INSTRUMENTAL_API_KEY", // REQUIRED
```

Comment:
I think we should be specific at project vs user token in these examples. Also, I think we've standardized on the term "token" through UX and upgraded docs. So `PROKECT_API_TOKEN`?